src/components/Board.vue

```diff
@@ -21,34 +19,33 @@
 import Vue from 'vue'
 import { Component } from 'vue-property-decorator'
 import Arrow from '@/gameComponents/Arrow.vue'
-import { EDirection, IFlowProps } from '@/types'
+import { EDirection, IAreasPositions } from '@/types'
 import { StoreModuleEnum } from '@/store/types'
 import { EActionScore } from '@/store/modules/score/typesScore'
 import { EActionArrow, IArrowData } from '@/store/modules/arrow/typesArrow'
 import { getRandom } from '@/helpers/getRandomHelper'
+import Areas from '@/gameComponents/Areas.vue'
 
 @Component({
   components: {
-    Arrow
+    Arrow, Areas
   }
 })
 export default class Board extends Vue {
   isActive: boolean = false // flag for start and finish game
   safeLoop: number = 0
+  areasPositions!: IAreasPositions
+
+  get positions (): IAreasPositions {
```

Comment:
Do you need this getter? Can you just use `areasPositions`?

Reply:
@silchencko I'll check it out. Thanks